Server/VikingVault.Services/BankAccountService.cs

```diff
@@ -46,9 +46,9 @@ public List<BankAccount> GetBankAccounts(string token)
             return bankAccounts;
         }
 
-        public BankAccount ChangeBalance(string email, UpdateBankAccountModel updatedBankAccount)
+        public BankAccount ChangeBalance(UpdateBankAccountModel updatedBankAccount)
```

Comment:
This method is named ChangeBalance but it only increments the value from the balance.

Reply:
It could also decrement it.